aiidalab_widgets_base/process.py

```diff
@@ -96,26 +93,11 @@ def __init__(  # pylint: disable=too-many-arguments
                 f"process_class argument must be a sublcass of {Process}, got {process_class}"
             )
 
-        # Handling the deprecation.
-        if inputs_generator is None and input_dictionary_function is None:
+        # Check if inputs generator is provided.
+        if inputs_generator is None:
             raise ValueError("The `inputs_generator` argument must be provided.")
-        if inputs_generator and input_dictionary_function:
-            raise ValueError(
-                "You provided both: `inputs_generator` and `input_dictionary_function` "
-                "arguments. Please provide `inpust_generator` only."
-            )
-        if input_dictionary_function:
-            inputs_generator = input_dictionary_function
-            warnings.warn(
-                (
-                    "The `input_dictionary_function` argument is deprecated and "
-                    "will be removed in the release 1.1 of the aiidalab-widgets-base package. "
-                    "Please use the `inputs_generator` argument instead."
-                ),
-                DeprecationWarning,
-            )
 
-        # Checking if the inputs generator is
+        # Checking if the inputs generator is callable
```

Comment:
I'd recommend the following sequence:
```python
if callable(inputs_generator):
    self.inputs_generator = inputs_generator
else:
    raise ValueError("The `inputs_generator` argument must be provided and a function that ...")
```

Reply:
I don't understand what you mean, the code you paste here is all the same.

Reply:
You have an additional `is None` check which is unnecessary I think (`None` is not `callable`).

Reply:
Thanks! I see. I remove the `None` as the default value of `inputs_generator` argument.